services/subscriber/service.go

```diff
@@ -324,8 +324,10 @@ func (s *Service) newPointsWriter(u url.URL) (PointsWriter, error) {
 	switch u.Scheme {
 	case "udp":
 		return NewUDP(u.Host), nil
-	case "http", "https":
+	case "http":
 		return NewHTTP(u.String(), time.Duration(s.conf.HTTPTimeout))
+	case "https":
```

Comment:
Could we add a log line here if InsecureSkipVerify is true? This way we can warn the operator that an insecure setting has been used.